server/controllers/comment.controller.js

```diff
@@ -105,17 +109,16 @@ const updateComment = async (req, res) => {
     await CommentHistory.create({
       comment_id: res.locals.comment.id,
       body: res.locals.comment.body,
-      is_updated: true,
     });
   }
 
   res.status(200).json({
-    editedComment: editComment,
+    editedComment: editComment[1],
```

Comment:
Any reason why we are returning the second element in the `editComment` array?

Reply:
The patch returns the amount of rows affected by the action. So, I didn't want to include that, hence why I brought out only the second element.
<img width="567" alt="Screenshot 2019-04-17 at 8 40 18 AM" src="https://user-images.githubusercontent.com/32884331/56269909-cb3f9180-60ec-11e9-8591-0825d784feb8.png">